server/js_test.go

```diff
@@ -0,0 +1,122 @@
+package server
```

Comment:
Was this file added by error? Not sure the purpose of it (and staticcheck complains that withJetStream is never used):
```
server/js_test.go:11:6: func withJetStream is unused (U1000)
```

Reply:
yes, doh. Will remove